nebd/test/part1/fake_file_service.cpp

```diff
@@ -35,11 +35,11 @@ void FakeNebdFileService::OpenFile(::google::protobuf::RpcController* controller
     brpc::Controller* cntl = static_cast<brpc::Controller*>(controller);
 
     LOG(INFO) << "logid = " << cntl->log_id() << ", OpenFile.";
-    if (buffer == nullptr) {
-        response->set_retcode(RetCode::kNoOK);
-        response->set_retmsg("OpenFile FAIL");
-        return;
-    }
+    // if (buffer == nullptr) {
```

Comment:
why remove these code?

Reply:
buffer is a global variable, it cannot be nullptr.

Reply:
https://github.com/opencurve/curve/blob/48d81bb78283e36827821dad7255b829455a2208/nebd/test/part1/fake_file_service.cpp#L28